### PR TITLE
Add info about storing remember me tokens in a database

### PR DESCRIPTION
**Pull request by pierredup**
I only recently saw that the remember me functionality has a `token_provider` option, and there is a default class to store tokens in the database as part of the DoctrineBridge, however there is no clear information on how this provider should be used (the class itself contains some hints, but the `token_provider` setting wasn't clear which values it accepts).
So I want to add a section to the remember me docs to explain how to use the `DoctrineTokenProvider`, which might also give some hints on how to create a custom provider

### CODE REVIEW

File: security/remember_me.rst [hunk 1]
```diff
@@ -261,3 +257,116 @@ your controller using annotations::
 
 For more information on securing services or methods in this way,
 see :doc:`/security/securing_services`.
+
+.. _token_in_database:
+
+Storing remember me tokens in the database
```
Comment (OskarStark) [inline security/remember_me.rst]: Please use:

Storing Remember Me Tokens in the Database

File: security/remember_me.rst [hunk 2]
```diff
@@ -261,3 +257,116 @@ your controller using annotations::
...
 For more information on securing services or methods in this way,
 see :doc:`/security/securing_services`.
+
+.. _token_in_database:
+
+Storing Remember Me Tokens in the Database
+------------------------------------------
+
+By default, tokens are stored in a cookie. You can choose to store the token in a database,
+to not have a (hashed) version of the password in a cookie.
```
Comment (noniagriconomie) [inline security/remember_me.rst]: why (hashed) with parenthesis? 

also, is it really the password? or like an hash?

Comment (pierredup) [reply security/remember_me.rst]: > why (hashed) with parenthesis?

This was just copied from the original description, I'm not sure why the parenthesis is needed

> also, is it really the password? or like an hash?

It's a hashed version of the password

**Comment by javiereguiluz** [comment]
@pierredup I didn't know about this feature ... so thanks a lot for contributing these docs and for teaching me about this!

**Pull request merged**